From 090a42caf768b607bf32d09e65ea8af1cebee0c0 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 27 Feb 2018 12:03:30 +0800
Subject: [PATCH] arm64: rockchip: px30: Remove the initial value of PLL_CPLL
 and BOOST

If show logo in uboot, can't change vop clocks.

Change-Id: I84fc1138b54b1c7b3c798f1bc4fb7d0f332e6895
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/px30.dtsi | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/px30.dtsi b/arch/arm64/boot/dts/rockchip/px30.dtsi
index acb3643b7651..993fdf8076a5 100644
--- a/arch/arm64/boot/dts/rockchip/px30.dtsi
+++ b/arch/arm64/boot/dts/rockchip/px30.dtsi
@@ -809,14 +809,8 @@
 		#clock-cells = <1>;
 		#reset-cells = <1>;
 
-		assigned-clocks =
-			<&cru APLL_BOOST_H>, <&cru APLL_BOOST_L>,
-			<&cru PLL_NPLL>, <&cru PLL_CPLL>,
-			<&cru ARMCLK>;
-		assigned-clock-rates =
-			<1608000000>, <1416000000>,
-			<1188000000>, <1188000000>,
-			<816000000>;
+		assigned-clocks = <&cru PLL_NPLL>, <&cru ARMCLK>;
+		assigned-clock-rates = <1188000000>, <816000000>;
 	};
 
 	pmucru: pmu-clock-controller@ff2bc000 {
-- 
2.35.3

